node/cmd/guardiand/admintemplate.go

```diff
@@ -495,6 +527,103 @@ func runRecoverChainIdTemplate(cmd *cobra.Command, args []string) {
 	fmt.Print(string(b))
 }
 
+func runAccountantModifyBalanceTemplate(cmd *cobra.Command, args []string) {
+	if *accountantModifyBalanceModule == "" {
+		log.Fatal("--module must be specified.")
+	}
+	if *accountantModifyBalanceTargetChainId == "" {
+		log.Fatal("--target-chain-id must be specified.")
+	}
+	targetChainID, err := parseChainID(*accountantModifyBalanceTargetChainId)
+	if err != nil {
+		log.Fatal("failed to parse target chain id: ", err)
+	}
+	if *accountantModifyBalanceSequence == "" {
+		log.Fatal("--sequence must be specified")
+	}
+	sequence, err := strconv.ParseUint(*accountantModifyBalanceSequence, 10, 64)
+	if err != nil {
+		log.Fatal("failed to parse sequence as uint64: ", err)
+	}
+	if *accountantModifyBalanceChainId == "" {
+		log.Fatal("--chain-id must be specified.")
+	}
+	chainID, err := parseChainID(*accountantModifyBalanceChainId)
+	if err != nil {
+		log.Fatal("failed to parse chain id: ", err)
+	}
+	if *accountantModifyBalanceTokenChainId == "" {
+		log.Fatal("--token-chain-id must be specified.")
+	}
+	tokenChainID, err := parseChainID(*accountantModifyBalanceTokenChainId)
+	if err != nil {
+		log.Fatal("failed to parse token chain id: ", err)
+	}
+	if *accountantModifyBalanceTokenAddress == "" {
+		log.Fatal("--token-address must be specified.")
+	}
+	tokenAddress, err := parseAddress(*accountantModifyBalanceTokenAddress)
+	if err != nil {
+		log.Fatal("failed to parse token address: ", err)
+	}
+	if *accountantModifyBalanceKind == "" {
+		log.Fatal("--action must be specified")
+	}
+	action, err := strconv.ParseUint(*accountantModifyBalanceKind, 10, 8)
+	if err != nil {
+		log.Fatal("failed to parse modification action as uint64: ", err)
+	}
+	if action != uint64(nodev1.ModificationKind_MODIFICATION_KIND_ADD) && action != uint64(nodev1.ModificationKind_MODIFICATION_KIND_SUBTRACT) {
+		log.Fatal("invalid modification action, must be 1 (add) or 2 (subtract)")
+	}
+	if *accountantModifyBalanceAmount == "" {
+		log.Fatal("--amount must be specified.")
+	}
+	amount_big := big.NewInt(0)
+	amount_big, ok := amount_big.SetString(*accountantModifyBalanceAmount, 10)
+	if !ok {
+		log.Fatal("failed to parse amount")
+	}
+	_, overflow := uint256.FromBig(amount_big)
+	if overflow {
+		log.Fatal("amount overflowed uint256")
+	}
+	if *accountantModifyBalanceReason == "" {
+		log.Fatal("--reason must be specified.")
+	}
+	if len(*accountantModifyBalanceReason) > 32 {
+		log.Fatal("reason is too long, can be at most 32 bytes")
```

Comment:
Can any or all of these numbers be constants? Should they be?

Reply:
I added a length for the reason. I think that's the only one here for which that makes sense. Could maybe do another PR some day to add consts for all the other commands in `sdk/vaa/payload.go`, but that's beyond the scope of this PR.